package.json

```diff
@@ -29,10 +31,14 @@
     "eslint-plugin-vue": "^6.2.2",
     "lint-staged": "^9.5.0",
     "node-sass": "^4.12.0",
+    "parcel-bundler": "^1.12.4",
```

Comment:
Is another bundler (alongside webpack) needed in this project? What is it used for?